source/clear-linux/tutorials/kubernetes/kubernetes.rst

```diff
@@ -150,45 +150,37 @@ Configure and run CRI-O + kata-runtime
        sudo systemctl daemon-reload
        sudo systemctl restart crio
 
-#.  Initialize the master control plane with the command:
+#.  Initialize the master control plane with the command below and follow the printed instructions to set up `kubectl`:
```

Comment:
Suggest changing from "follow the printed" to "follow the displayed".

Reply:
done.